server/views/components/actions-card/macro.njk

```diff
@@ -1,3 +1,3 @@
-{% macro actionsCard(actions) %}
+{% macro actionsCard(params) %}
```

Comment:
I've named the argument `params` now, so that it is consistent with other GDS components; and also it now contains more that just the array of actions 👍